api/core/model_runtime/model_providers/tongyi/llm/qwen-max-1201.yaml

```diff
@@ -8,54 +8,68 @@ model_properties:
 parameter_rules:
   - name: temperature
     use_template: temperature
-    default: 1.0
-    min: 0.0
+    type: float
+    default: 0.85
+    min: 0.01
```

Comment:
```suggestion
    min: 0.0
```
Leave the 0 as minimum.

Reply:
![image](https://github.com/langgenius/dify/assets/1500781/ffc64c88-cafa-4ee8-95e6-2253d3a4d64d)
Thanks for pointing out that it is set to 0.01 here because the official said 0 is meaningless.

Reply:
But 0 IS an allows value for temperature as the lower limit shown in your docs.
And `0.01` brings an precision discussion here

Reply:
okay, done~

Reply:
Cool, thanks. And it brings consistency with Qwen and other existed models.